src/pt/spectralscan/src/eu/kanade/tachiyomi/extension/pt/spectralscan/NexusScan.kt

```diff
@@ -0,0 +1,311 @@
+    }
+
+    // ==================== Details =======================
+
+    override fun mangaDetailsParse(response: Response): SManga {
+        val document = response.asJsoup()
+
+        return SManga.create().apply {
+            title = document.selectFirst("h1.item-title, h1")?.text()?.trim() ?: ""
+            thumbnail_url = document.selectFirst("img.item-cover-image")?.absUrl("src")
+            description = document.select(".item-description p, .item-description").text()
+
+            genre = document.select(".genre-tag, .hero-genres .genre-tag").joinToString { it.text().trim() }
+
+            // Procura autor/artista nos detalhes
+            document.select(".detail-item").forEach { item ->
+                val label = item.selectFirst(".detail-label")?.text()?.lowercase() ?: ""
+                val value = item.selectFirst(".detail-value")?.text()?.trim()
+                when {
+                    label.contains("autor") || label.contains("author") -> author = value
+                    label.contains("artista") || label.contains("artist") -> artist = value
+                    label.contains("status") -> {
+                        status = when (value?.lowercase()) {
+                            "em andamento", "ongoing", "ativo", "lançando" -> SManga.ONGOING
+                            "completo", "completed", "finalizado" -> SManga.COMPLETED
+                            "pausado", "hiato", "on hiatus" -> SManga.ON_HIATUS
+                            "cancelado", "cancelled", "dropped" -> SManga.CANCELLED
+                            else -> SManga.UNKNOWN
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    // ==================== Chapter =======================
+
+    private fun getMangaSlug(url: String) = url.substringAfter("/manga/").trimEnd('/')
+
+    override fun fetchChapterList(manga: SManga): Observable<List<SChapter>> {
+        return client.newCall(chapterListRequest(manga)).asObservableSuccess()
+            .map { it.parseAs<ChapterListResponse>() }
+            .flatMap { firstPage ->
+                val chapters = parseChaptersFromHtml(firstPage.chapters_html)
+
+                if (!firstPage.has_next) {
+                    Observable.just(chapters)
+                } else {
+                    fetchRemainingChapters(getMangaSlug(manga.url), chapters, 2)
+                }
+            }
+    }
+
+    private fun fetchRemainingChapters(
+        slug: String,
+        accumulatedChapters: List<SChapter>,
+        currentPage: Int,
+    ): Observable<List<SChapter>> {
+        val url = "$baseUrl/ajax/load-chapters/".toHttpUrl().newBuilder()
+            .addQueryParameter("item_slug", slug)
+            .addQueryParameter("page", currentPage.toString())
+            .addQueryParameter("sort", "desc")
+            .addQueryParameter("q", "")
+            .build()
+
+        return client.newCall(GET(url, headers)).asObservableSuccess()
```

Comment:
Don't you also need `ajaxHeaders()` here? You're using the `/ajax/` endpoint.
```suggestion
        return client.newCall(GET(url, ajaxHeaders())).asObservableSuccess()
```